android/src/main/java/com/pspdfkit/flutter/pspdfkit/PspdfkitPlugin.java

```diff
@@ -76,16 +77,16 @@ private PspdfkitPlugin(Registrar registrar) {
      * Plugin registration.
      */
     public static void registerWith(Registrar registrar) {
-        final MethodChannel channel = new MethodChannel(registrar.messenger(), "pspdfkit");
+        MethodChannel channel = new MethodChannel(registrar.messenger(), "pspdfkit");
```

Comment:
This should stay `final`, as it is never changed. Also, the same applies to the `pspdfkitPlugin` variable below.